signing/secp256k1/impl/src/main/java/tech/pegasys/signers/secp256k1/EthPublicKeyUtils.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.signers.secp256k1;
+
+import static org.bouncycastle.util.BigIntegers.asUnsignedByteArray;
+
+import java.math.BigInteger;
+import java.security.AlgorithmParameters;
+import java.security.KeyFactory;
+import java.security.NoSuchAlgorithmException;
+import java.security.interfaces.ECPublicKey;
+import java.security.spec.ECGenParameterSpec;
+import java.security.spec.ECParameterSpec;
+import java.security.spec.ECPoint;
+import java.security.spec.ECPublicKeySpec;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.InvalidParameterSpecException;
+
+import org.apache.tuweni.bytes.Bytes;
+import org.apache.tuweni.bytes.Bytes32;
+import org.web3j.utils.Numeric;
+
+public class EthPublicKeyUtils {
+  private static final int PUBLIC_KEY_SIZE = 64;
+
+  public static ECPublicKey createPublicKey(final ECPoint publicPoint) {
+    try {
+      AlgorithmParameters parameters = AlgorithmParameters.getInstance("EC");
```

Comment:
static class level variable?

Reply:
It can't be because it throws a AlgorithmNotFoundException